libs/utils/gcp-auth-secret-manager/src/lib/gcp-auth-secret-manager.ts

```
@@ -10,7 +10,8 @@
  */
 
 import { SecretManagerServiceClient } from '@google-cloud/secret-manager';
-import { GcpInitializationError, getProjectId } from '@dulce/gcp-core';
+// Cache clients to avoid creating multiple instances
+const clientCache = new Map<string, SecretManagerServiceClient>();
 
 // Cache clients to avoid creating multiple instances
 const clientCache = new Map<string, any>();
```

Biome (2.1.2)

[error] Shouldn't redeclare 'clientCache'. Consider to delete it or rename it.

'clientCache' is defined here:


(lint/suspicious/noRedeclare)